src/routes/webhooks/paddle.ts

```diff
@@ -460,18 +476,325 @@ export const processGiftedPayment = async ({
   );
 };
 
+const checkTransactionStatusValid = ({
+  event,
+  transaction,
+  nextStatus,
+  validStatus,
+  data,
+}: {
+  event: EventEntity;
+  transaction: UserTransaction;
+  nextStatus: UserTransactionStatus;
+  validStatus: UserTransactionStatus[];
+  data: ReturnType<typeof getPaddleTransactionData>;
+}): boolean => {
+  if (!validStatus.includes(transaction.status)) {
+    logger.warn(
+      {
+        eventType: event.eventType,
+        provider: SubscriptionProvider.Paddle,
+        currentStatus: transaction.status,
+        nextStatus,
+        data,
+      },
+      'Transaction with invalid status',
+    );
+
+    return false;
+  }
+
+  return true;
+};
+
 export const processTransactionCompleted = async ({
   event,
 }: {
   event: TransactionCompletedEvent;
 }) => {
+  await notifyNewPaddleTransaction({ event });
```

Comment:
Let's update this function to differentiate between plus and cores so we will know from slack what the user purchased

Reply:
Yep, gonna update

Reply:
Added 0375c52b8755b4e4ffa2caf3ad8608fa8e848001